docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
+    command: docker/dev/run.sh
+    environment:
+      PORT: 5000
+      DATABASE_URL: postgres://log_puller:log_puller@postgres/log_puller_development
+      PG_SSL: 'false'
```

Comment:
At first I was surprised these booleans were contained in strings and then I remembered that all environment variables are strings. :)

Reply:
Actually, it was docker-compose complaining about them and I had to add these quotes. It doesn't seem to care if the string doesn't look like boolean (e.g. foo or database URL). Although I'm not a YML expert. ;)